src/Listener/FailedStepListener.php

```diff
@@ -57,14 +52,12 @@ class FailedStepListener implements EventSubscriberInterface
      * @param Mink $mink
      * @param string $logDirectory
      * @param boolean $screenshot
```

Comment:
Don't you like to have it aligned into two columns? 
```php
      * @param Mink      $mink
      * @param string     $logDirectory
      * @param boolean $screenshot
```
Or do you just want to safe the space? ;)